src/main/java/com/user/mngmnt/controller/UserController.java

```diff
@@ -74,6 +74,7 @@ public ModelAndView searchByTerm(@RequestParam(value = "page", defaultValue = "0
                                      @RequestParam(value = "searchTerm", required = false) String searchTerm) {
         ModelAndView modelAndView = new ModelAndView();
         modelAndView.setViewName("home");
+        System.out.println("Sajana Vijayan");
```

Comment:
code readability: you may put the System.out.println after modelAndView.addObject("maxTraySize", size);